internal/caravan/tradeagreement.lua

```diff
@@ -35,4 +99,14 @@ function TradeAgreementOverlay:init()
             on_activate=diplomacy_toggle_cat,
         },
     }
+    self:addviews {
+        widgets.HotkeyLabel {
+            frame = { t = 1, l = 0 },
+            label = 'Select gems at least as valuable as',
```

Comment:
Instead of "select at least as valuable as", how about "Select by value"? Only items with the shown value would be toggled on click. This would also be relevant beyond gems. Being able to select the cheapest leather, for example, is a common request.